Task 3/system/Coding/src/app/Parameters.h

```diff
@@ -11,37 +11,47 @@
 #define PARAMETERS_H
 
 /* INCLUDES ***************************************************************************************/
+#include "Types.h"
+
+#include "Display.h"
 
 /* CONSTANTS **************************************************************************************/
+#define INDEX_SET1 (0U)
+#define INDEX_SET2 (1U)
+#define INDEX_SET3 (2U)
 
 /* MACROS *****************************************************************************************/
 
 /* TYPES ******************************************************************************************/
+
 /** @struct ParameterSet
  * @brief struct to represent a ParameterSet
 */
 typedef struct
 {
-
+    Float32 kp;
+    Float32 kd;
+    Float32 ki;
+    Int16 motorspeed;
 }ParameterSet;
 
+/** Pointer to the selected parameter set */
+extern ParameterSet* pParameters;
+
 /* PROTOTYPES *************************************************************************************/
-/** 
+/**
  * @brief sets the selected ParameterSet
- * 
 */
-void Parameters_setParameterSet(void);
+extern void Parameters_setParameterSet(UInt8 index);
 
-/** 
- * @brief gets the current ParameterSet
- * 
+/**
+ * @brief displays the ParameterSet on the OLED-Display
```

Comment:
Use imperative form for doxygen -> this applies to all doxygen comments descriptions

```suggestion
 * @brief Display the ParameterSet on the OLED-Display.
```